app/src/main/java/com/karhoo/samples/uisdk/dropin/MainActivity.kt

```diff
@@ -154,9 +159,30 @@ class MainActivity : AppCompatActivity() {
         }
     }
 
+    var requestedAuthentication = false
     private fun applyAdyenTokenExchangeConfig() {
         val config = AdyenTokenExchangeConfig(applicationContext)
         config.paymentManager = adyenPaymentManager
+        config.externalAuthenticationNeeded = {
+            if (!requestedAuthentication) {
+                Log.e("matei", "Need an external authentication")
```

Comment:
I see you signed your work @mateid-kr :))

Reply:
It's still in development 😄 . The final PR won't contain these